src/indexPage/index.js

```diff
@@ -8,17 +8,43 @@ import { isEuCountry, showHideEuElements } from '../common/footer-checks';
 import GTM from '../common/gtm';
 import { load as loadLang, showBanner } from '../common/lang';
 import { moveToDeriv } from '../common/utils/utility';
-import { getTokenList } from '../common/utils/storageManager';
+import { get as getStorage, set as setStorage, remove, getTokenList } from '../common/utils/storageManager';
 import { createUrl } from '../common/utils/tools';
 import '../common/binary-ui/dropdown';
+import BotLanding from './react-components/BotLanding.jsx';
 
 const renderElements = () => {
-    ReactDOM.render(<Logo />, document.getElementById('binary-logo'));
-    ReactDOM.render(<Footer />, document.getElementById('footer'));
-    isEuCountry().then(isEu => showHideEuElements(isEu));
-    showBanner();
+    const today = new Date().getTime();
+    const expirationDate = today + 1000 * 60 * 60 * 24 * 21;
 
-    $('#shop-url').attr('href', createUrl({ subdomain: 'shop', path: 'collections/strategies', isNonBotPage: true }));
+    const temp = getStorage('setDueDateForBanner');
+
+    const elements = document.querySelectorAll('#notification-banner,#main,#footer,#header,#topbar');
+    if (!temp) {
+        ReactDOM.render(<BotLanding />, document.getElementById('bot-landing'));
+        setStorage('setDueDateForBanner', expirationDate);
+        for (let i = 0; i < elements.length; i++) {
```

Comment:
Does it support ES6 ? so we can write like: 
for(el of elements){
....
}

Reply:
[
![Screenshot 2022-04-27 at 12 53 36 PM (2)](https://user-images.githubusercontent.com/97010868/165443556-fe2e02ee-2b17-4d60-8ebc-107f2a634704.png)
](url)

Reply:
getting this error when i used for of loop